packages/migrator/readme.md

```diff
@@ -46,7 +46,7 @@ npm install --save-dev @slonik/migrator
 
 Then in a file called `migrate.js`:
 ```javascript
-const {SlonikMigrator} = require('@slonik/migrator')
+const {SlonikMigrator, prettyLogger}} = require('@slonik/migrator')
```

Comment:
What do you think about making `prettyLogger` a static prop on SlonikMigrator instead? That way only one import is needed.

Reply:
Excellent idea, that's done now.